src/autoimport/model.py

```diff
@@ -366,8 +368,13 @@ def _find_package_in_common_statements(name: str) -> Optional[str]:
         Returns:
             import_string
         """
-        if name in common_statements:
-            return common_statements[name]
+        if "common_statements" in self.config:
+            local_common_statements = self.config["common_statements"]
+        else:
```

Comment:
Wouldn't it be better, if instead of replacing the default common statements, they overwrite them?

I'm thinking in the case that a user likes the default ones, but wants to add or overwrite some of them.

If you think it's not a probable case, I'm fine with leaving it as is

Reply:
It makes sense that users might want to overwrite some of the defaults instead of replacing the default `common_statements` entirely.

The downside of overwriting the default `common_statements` instead of replacing is that then there is no way to *remove* keys from the `common_statements` dict.
Maybe this is not such a big deal though :)

Reply:
Updated in https://github.com/lyz-code/autoimport/pull/160/commits/59c1be2aff38d7b3c8fc319e31ff30e84426ff1c.